content/influxdb/cloud-dedicated/reference/sql/functions/time-and-date.md

```diff
@@ -360,6 +362,279 @@ GROUP BY _time, room
 {{% /expand %}}
 {{< /expand-wrapper >}}
 
+## date_bin_wallclock
+
+Calculates time intervals using the timezone of a specified time value and
+returns the start of the interval nearest to the specified timestamp.
+Use `date_bin_wallclock` to downsample time series data by grouping rows into
+time-based "bins" or "windows" that are based off "wall clock" times in a
+specific timezone and applying an aggregate or selector function to each window.
+
+### Time zone shifts
+
+Many regions use time zone shifts (such as Daylight Saving Time (DST) and British
+Summer Time (BST)). If a wall clock time bin starts at a time that does not
+exist in the specified time zone, the timestamp is adjusted to the time that is
+the same offset from the start of the day in that time zone.
+
+If a wall clock time represents an ambiguous time in the region then the
+behavior depends on the size of the specified interval. If the interval is
+larger than the difference between the two possible timestamps, then the earlier
+timestamp is used. Otherwise, the function uses the timestamp that matches the
+UTC offset of the input timestamp.
+
+```sql
+date_bin_wallclock(interval, expression[, origin])
+```
+
+##### Arguments:
+
+- **interval**: Bin interval.
+- **expression**: Time expression to operate on.
+  Can be a constant, column, or function.
+  The output timestamp uses the time zone this time expression.
+- **origin_timestamp**: Starting point used to determine bin boundaries.
+  This must be a "wall clock" timestamp (no time zone).
+  _Default is the Unix epoch._
+
+The following intervals are supported:
+
+- nanoseconds
+- microseconds
+- milliseconds
+- seconds
+- minutes
+- hours
+- days
+- weeks
+
+> [!Note]
+> `date_bin_wallclock` does _not_ support month-, year-, or century-based intervals.
+
+{{< expand-wrapper >}}
+{{% expand "View `date_bin_wallclock` query example" %}}
+
+_The following examples ._
+
+The following query uses  the sample data set provided in the
+[Get started with InfluxDB tutorial](/influxdb/cloud-dedicated/get-started/write/#construct-line-protocol)
+and returns the 12-hour average temperature for each room using times in the
+`America/Los_Angeles` time zone.
+
+{{% influxdb/custom-timestamps %}}
+
+```sql
+SELECT
+  date_bin_wallclock(INTERVAL '12 hours', tz(time, 'America/Los_Angeles')) AS time,
```

Comment:
Potential problem using "time" as the alias?

Reply:
No, because I'm using `1` in the `GROUP BY` clause, so it doesn't base grouping off the alias, just the column position.